ckrack/optimus-bundle/0.1/config/packages/ckrack_optimus.yaml

```diff
@@ -0,0 +1,13 @@
+ckrack_optimus:
+
+    # Large prime number lower than 2147483647
```

Comment:
Is a prime considered "large" if it is smaller than 2^31?

Isn't RSA using a few thousand bits log primes?

Reply:
I think what's meant is, large but lower than 2^31.
Anyway, it's a direct quote from https://github.com/jenssegers/optimus#usage

I'll probably replace the comments in env with an explanation to generate via the spark command.